### ISSUE TITLE
Can not found `libwx_*` path

### ISSUE THREAD

**Issue by ByeongUkChoi** [elixir-desktop/deployment#2]
my mac version is 12.6 (monterey)

I can't find wxWidgets path in here:
https://github.com/elixir-desktop/deployment/blob/main/lib/package.ex#L92
I think this path is wrong. How can I do?

**Comment by dominicletz** [comment]
Yes, the path is still hardcoded. It must be something smarter here for the general use case. Not sure how to best detect the current wxWidgets library directory

**Comment by djgoku** [comment]
I was going to create an issue and not sure if this is related to my issue.

@dominicletz thanks for this and all associated packages!

I encounter an error while attempting to execute my built MyApp.app on a clean test system running macOS. However, I can resolve this issue by installing wxWidgets on the test system.

I shouldn't need to install wxWidgets right? Is there an easy way to add this during build?

```
(Failed to load NIF library: 'dlopen(/Applications/MyApp.app/Contents/Resources/lib/wx-2.4.3/priv/wxe_driver.so, 0x0002): Library not loaded: /nix/store/d7xy60nv22w594gwvahyfwl7bpzj7571-wxwidgets-3.2.6/lib/libwx_osx_cocoau_stc-3.2.0.dylib)
```
